src/plugins/intel_cpu/src/emitters/plugin/x64/jit_load_store_emitters.cpp

```diff
@@ -1283,7 +1283,10 @@ void jit_store_emitter::store_dword_to_word_extension(const Xbyak::Reg64& reg,
             if (src_prc_ == ov::element::f32) {
                 ymm = Ymm(aux_vec_idxs[0]);
             }
-            uni_vcvtneps2bf16_->emit_code({static_cast<size_t>(zmm.getIdx())}, {static_cast<size_t>(ymm.getIdx())});
+            uni_vcvtneps2bf16_->emit_code({static_cast<size_t>(zmm.getIdx())},
+                                          {static_cast<size_t>(ymm.getIdx())},
+                                          {},
+                                          {});
```

Comment:
After new implementation these changes are not necessary anymore.
Please double check all the updates

Reply:
Done